news/3386.trivial.rst

```diff
@@ -0,0 +1 @@
+Decouple project from the existence of Pipfile.
```

Comment:
Can you make this more descriptive, e.g. "Fixed a bug with <blah blah blah> so that <blah blah blah>"? I can't really tell from this exactly why this is good.  This is going in a changelog entry and I doubt anyone cares that these were coupled or knows what it means that they aren't anymore

Reply:
Thanks, I updated the news files.